nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITDirectory.java

```diff
@@ -100,4 +102,33 @@ public void itCanCreateUpdateFetchDeleteDirectory() {
         directory.deleteEntry(result.getId());
     }
 
+    @Test
+    public void itCanSetOrderingPropertyType() {
```

Comment:
Could you split this test in two, one for the Integer way and another for the Long way?